apps/organisations/migrations/0023_alter_organisationtranslation_information.py

```diff
@@ -0,0 +1,22 @@
+# Generated by Django 3.2.19 on 2024-03-14 12:05
+
+from django.db import migrations
+import django_ckeditor_5.fields
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("a4_candy_organisations", "0022_ckeditor_iframes"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="organisationtranslation",
```

Comment:
hmm, this is a generated PR, and it is fetching the `organisation` model as `organisationtranslation`. Should we do the same in the 0022?
@goapunk

Reply:
I don't see the OrganisationTranslation model, which is also imported in the forms.py from the models.py

Reply:
It's probably created by parler?